### ISSUE TITLE
Remove white background colour of output on Slider component

### ISSUE THREAD

**Issue by canxerian** [equinor/design-system#3105]
The output element on the Slider component has a background colour of white. This poses an issue when it's used on non-white backgrounds, such as:

![slider background color](https://github.com/equinor/design-system/assets/1138440/a83e9b36-97f3-4a7c-9d07-b816c55bbe15)

![image](https://github.com/equinor/design-system/assets/1138440/1a4552e7-1151-480c-bcfa-eaf8cb3797b6)

**Steps to reproduce the bug**
Instantiate a slider,
```tsx
<Slider value={4} min={0} max={10} aria-labelledby="simple-slider" />
```
Slide the slider value.

**Comment by oddvernes** [comment]
I see! Well the background is there so the value doesn't collide with the min/max numbers. I think we may need to rethink this design and come up with something else, like have the value showing as a tooltip, optionally only on hover, and perhaps above instead of below. [How it's done in mantine](https://mantine.dev/core/slider/). [How it's done in MUI](https://mui.com/material-ui/react-slider/).

**Comment by canxerian** [comment]
The tooltip 

> w. [How it's done in mantine](https://mantine.dev/core/slider/). [How it's done in MUI](https://mui.com/material-ui/react-slider/).



> [How it's done in mantine](https://mantine.dev/core/slider/). [How it's done in MUI](https://mui.com/material-ui/react-slider/).

These are both very nice examples

**Comment by BirteThornquist** [comment]
Redesigned with the value in a tooltip placed above the tracking line:

![Image](https://github.com/equinor/design-system/assets/74531659/f1ebeee9-865b-4542-ae75-64ad8f4c273b)

**Comment by BirteThornquist** [comment]
Removed the dots on the track line as this is a new feature we don't have at the moment and will require some time to implement. The changes I have done is to add a tooltip above the handler dot that shows the value. I have also changed the text used as I think the text is too small. New text style used is caption text (14px) for both the dot and the tooltip text. [Here](https://www.figma.com/file/dCtGxSjiufV5C6YcXSs3MD/Birte-test?type=design&node-id=3046-753&mode=design&t=5PVcnLqoSX968wR1-4) is the design to be reviewed. 



![Image](https://github.com/equinor/design-system/assets/74531659/d88b0be1-1bb3-4217-96f3-14523a97c715)
